pkg/services/ngalert/schedule/schedule.go

```diff
@@ -139,18 +154,164 @@ func (sch *schedule) Unpause() error {
 }
 
 func (sch *schedule) Run(ctx context.Context) error {
-	sch.wg.Add(1)
+	sch.wg.Add(2)
 
 	go func() {
 		if err := sch.ruleEvaluationLoop(ctx); err != nil {
 			sch.log.Error("failure while running the rule evaluation loop", "err", err)
 		}
 	}()
 
+	go func() {
+		if err := sch.adminConfigSync(ctx); err != nil {
+			sch.log.Error("failure while running the admin configuration sync", "err", err)
+		}
+	}()
+
 	sch.wg.Wait()
 	return nil
 }
 
+// SyncAndApplyConfigFromDatabase looks for the admin configuration in the database and adjusts the sender(s) accordingly.
+func (sch *schedule) SyncAndApplyConfigFromDatabase() error {
+	sch.log.Info("start of admin configuration sync")
+	orgIds, err := sch.adminConfigStore.GetOrgsWithAdminConfiguration()
+	if err != nil {
+		return err
+	}
+
+	q := &models.GetOrgAdminConfiguration{OrgIDs: orgIds}
+	if err := sch.adminConfigStore.GetAdminConfigurations(q); err != nil {
+		return err
+	}
+
+	sch.log.Info("found admin configurations", "count", len(q.Result))
```

Comment:
Although experience has shown me that this can be quite useful with debugging, I agree that info might be too spammy for an optional feature - I'll move it to debug.

Reply:
Done.